book/src/user/supported-platforms.md

```diff
@@ -0,0 +1,48 @@
+# Platform Support
+
+Support for different platforms are organized into three tiers, each with a
+different set of guarantees. For more information on the policies for platforms
+at each tier, see the [Platform Tier Policy](platform-tier-policy.md).
+
+Platforms are identified by their "target triple" which is a string composed by
```

Comment:
```suggestion
Platforms are identified by their Rust "target triple" which is a string composed by
```

Reply:
Rust's target triples don't always line up with the LLVM or GCC ones though. We have to maintain an internal mapping from one to the other in the `zcashd` build scripts as we deal in both worlds, but we identify with the LLVM / GCC ones as that's our outermost compilation surface. `zebra` is primarily Rust, so it would be fine to focus on the Rust-specific targets in its case.

Reply:
That's correct. Although it's common to do `gcc -dumpmachine` to know your actual `target triplet` (which matches with LLVM/GCC). I'm not aware if Rust is _adapting_ those, as I've been used to the common structure, which is stated here https://wiki.osdev.org/Target_Triplet 

We can surely indicate we're focusing on Rust-specific targets, as I don't think we're going a different way if we can comply with Rust's standards.